lib/components/Card/Card.js

```diff
@@ -0,0 +1,413 @@
+!(function(e, t) {
```

Comment:
why are these files minified partially? e.g. names are all minified but the whitespace is not

Reply:
lol oops. that was my IDE. I was inspecting this file. I can rerun

Reply:
shouldnt matter in the long run. really the lib files should really not be minified (I could not figure out how to make webpack _not_ minify them)